src/python/impactx/dashboard/Analyze/plot_PhaseSpaceProjections/phaseSpace.py

```diff
@@ -53,7 +53,9 @@ def run_simulation():
     #   reference particle
```

Comment:
In a follow-up PR, `run_simulation()` really needs to be moved into a more well-named file than `phaseSpace.py` please :)

Reply:
Agree, addressed in https://github.com/ECP-WarpX/impactx/pull/725!